src/Bibliography.js

```diff
@@ -138,11 +138,14 @@ export default class Bibliography {
 
 		let parts = parseFormattedCitationSequence(text, citations);
```

Comment:
But now we don't need to run this first — we can simply branch.

Reply:
Good point! Thanks

Reply:
Done! Did I get your idea right?